hazelcast/src/main/java/com/hazelcast/jet/pipeline/Sources.java

```diff
@@ -1495,4 +1496,42 @@ public static <T> BatchSource<T> jdbc(
         return batchFromProcessor("jdbcSource",
                 SourceProcessors.readJdbcP(connectionURL, query, createOutputFn));
     }
+
+    /**
+     * Same as @{link {@link Sources#jdbc(String, String, FunctionEx)}}
+     * <p>
+     * This overload allows passing some properties for JDBC
+     * <p>
+     * Example for PostgreSQL fetchSize: <pre>{@code
+     *        Properties properties = new Properties();
+     *        properties.put(JdbcPropertyKeys.FETCH_SIZE, "5");
+     *        properties.put(JdbcPropertyKeys.AUTO_COMMIT, "false");
+     *        p.readFrom(Sources.jdbc(
+     *            DB_CONNECTION_URL,
+     *            "select ID, NAME from PERSON",
+     *            properties
+     *            resultSet -> new Person(resultSet.getInt(1), resultSet.getString(2))))
+     *    }</pre>
+     *    <p>
+     * Example for MySQL fetchSize: The DB_CONNECTION_URL should have "&useCursorFetch=true" parameter
```

Comment:
Maybe add something like:
> It's not always possible to use built-in properties, for example MySQL requires fetch size to be provided in the URL instead of connection properties.
+

DB_CONNECTION_URL -> database connection URL

+
replace constant below in snippet to some mock value, including parameter

Reply:
rev : e9a32b4e55050704a37efa5a9b0f778474bea336